projects/plugins/crm/src/entities/class-quote.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Quote Entity.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Entities;
+
+/**
+ * Quote class.
+ *
+ * @since $$next-version$$
+ */
+class Quote {
+
+	/**
+	 * The DB ID of the object.
+	 *
+	 * @var int
+	 */
+	public $id = -1;
+
+	/**
+	 * The owner of the object.
+	 *
+	 * @var int
+	 */
+	public $owner = -1;
+
+	/**
+	 * The quote meta.
+	 *
+	 * @var array
+	 */
+	public $meta = array();
+
+	/**
+	 * The quote tags.
+	 *
+	 * @var array
+	 */
+	public $tags = array();
+
+	/**
+	 * The quote files.
+	 *
+	 * @var array
+	 */
+	public $files = array();
+
+	/**
+	 * The quote notes.
+	 *
+	 * @var array
+	 */
+	public $notes = array();
```

Comment:
`$notes` should be a string.

Reply:
Copied that from the class_contact entity example directly 😛 That'll be the case in any of the other entity class PRs I've done so I'll check them too, thanks.